quicksort/QuickSort.java

```diff
@@ -0,0 +1,36 @@
+public class QuickSort {
+
+    static int partition(int[] a, int l, int r) {
```

Comment:
@jsroyal This means that specify inheritance level `public` or `private` before the `partition` method.